packages/react-devtools-shell/src/devtools.js

```diff
@@ -17,8 +17,17 @@ const {contentDocument, contentWindow} = iframe;
 // Helps with positioning Overlay UI.
 contentWindow.__REACT_DEVTOOLS_TARGET_WINDOW__ = window;
 
+// Initialize the backend and frontend early so that we have the messages that we
+// need during initial render
```

Comment:
Can you expand this comment a little bit? ("the messages we need" is a little vague)